src/app/mat-card/mat-card.component.ts

```diff
@@ -47,15 +44,18 @@ export class MatCardComponent implements OnInit {
 
       }
       else {
-        this.router.navigate(['/goshopping'])
+        path.navigate(['/goshopping'])
```

Comment:
wrong you are already in goshopping route why do you want to navigate again to same route....?

Reply:
> wrong you are already in goshopping route why do you want to navigate again to same route....?

sorry maam but everyone is having the same syntax .